### PR TITLE
docs: add main documentation files

### PR DESCRIPTION
**Pull request by vaeng**
I hope I hit a good middle ground between verbose and concise.

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
 # About
 
-<!-- TODO: write document
+[Uiua][uiua homepage] is an [array-][array-oriented programming] and [stack-oriented][stack-oriented programming] programming language that emphasizes concise, point-free code through [tacit programming][tacit programming].
+Its design is influenced by languages like [APL][apl-wiki] and J, which also prioritize powerful array manipulations.
+Uiua’s core operations are represented by symbols, and many built-in functions have symbolic representations.
+Function names like "pick" and "length" are automatically converted to their respective glyphs, `⊡` and `⧻`, without requiring special input from users.
```
Comment (ErikSchierboom) [inline docs/ABOUT.md:6]:
```suggestion
Function names like `pick` and `length` are automatically converted to their respective glyphs, `⊡` and `⧻`, without requiring special input from users.
```

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
 # About
 
-<!-- TODO: write document
+[Uiua][uiua homepage] is an [array-][array-oriented programming] and [stack-oriented][stack-oriented programming] programming language that emphasizes concise, point-free code through [tacit programming][tacit programming].
+Its design is influenced by languages like [APL][apl-wiki] and J, which also prioritize powerful array manipulations.
+Uiua’s core operations are represented by symbols, and many built-in functions have symbolic representations.
```
Comment (ErikSchierboom) [inline docs/ABOUT.md:5]:
```suggestion
Uiua's core operations are represented by symbols, and many built-in functions have symbolic representations.
```

File: docs/ABOUT.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+Function names like "pick" and "length" are automatically converted to their respective glyphs, `⊡` and `⧻`, without requiring special input from users.
 
-  This document contains a short introduction to the language.
+Created and maintained by [Kai Schmidt][kai schmidt github], Uiua is an open-source language - currently its experimental stages.
```
Comment (ErikSchierboom) [inline docs/ABOUT.md:8]: Maybe link to the uiua repository for the "open-source language" bit?

File: docs/ABOUT.md [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
+The language prioritizes stability in its core functions.
+Experimental features are marked, which our exercise framework generally avoids.
```
Comment (ErikSchierboom) [inline docs/ABOUT.md:10]: Maybe this line makes more sense in the learning document?

File: docs/INSTALLATION.md [hunk 5]
```diff
@@ -1,15 +1,20 @@
 # Installation
 
-<!-- TODO: write document
+[Pre-compiled Uiua binaries][releases] are available, so compiling isn’t necessary.
```
Comment (ErikSchierboom) [inline docs/INSTALLATION.md:3]:
```suggestion
[Pre-compiled Uiua binaries][releases] are available, so compiling isn't necessary.
```

File: docs/INSTALLATION.md [hunk 6]
```diff
@@ -1,15 +1,20 @@
...
+See the [official installation guide][install-guide] for the latest instructions including IDE integration and fonts.
```
Comment (ErikSchierboom) [inline docs/INSTALLATION.md:4]:
```suggestion
See the [official installation guide][install-guide] for the latest instructions, including IDE integration and fonts.
```

File: docs/RESOURCES.md [hunk 7]
```diff
@@ -1,13 +1,14 @@
...
-  Feel free to link to any good resources you know, whether they
-  be websites, videos, mailing lists, etc.
-
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/resources`.
-
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+- The most important resource is the [official documentation][docs]
+- For online editing, use the [Uiua Pad][pad]
+- Dive into the sourcecode on Uiua's [GitHub repository][github]
```
Comment (ErikSchierboom) [inline docs/RESOURCES.md:5]:
```suggestion
- Dive into the source code on Uiua's [GitHub repository][github]
```

File: docs/TESTS.md [hunk 8]
```diff
@@ -1,15 +1,43 @@
...
+Feel free to add additional test cases for experimenting.
+However, modifying tests does not affect the online test-runner process.
 
-  This document should describe everything related to running tests in the track.
+## Local Testing
 
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+After installing Uiua, run tests with `uiua test [PATH]`.
+To execute the test file, use `uiua tests tests.ua` in the terminal.
```
Comment (ErikSchierboom) [inline docs/TESTS.md:11]: Maybe shorten this to something like:

```suggestion
To runs an exercise's tests, open a terminal in the exercise's directory and run the `uiua tests tests.ua` command.
```

File: docs/TESTS.md [hunk 9]
```diff
@@ -1,15 +1,43 @@
...
+```bash
+$ uiua test tests.ua
+Test passed
+```
+
+## Skipped Tests
+
+Some exercises might include skipped tests to help you focus on early functionality before handling complex cases.
+Other exercises might exclude some test cases, the creator did not seem necessary to get a passing grade for the core of the concept being taught.
+To unskip tests, remove the relevant sections in the tests.ua file.
```
Comment (ErikSchierboom) [inline docs/TESTS.md:37]: We don't have this, do we?

**Review by ErikSchierboom — COMMENTED** [review]
Thanks for doing this! Couple of comments.

File: docs/TESTS.md (hunk 9 above)
Comment (vaeng) [reply docs/TESTS.md:37]: Not yet, but it might help with questions down the road. Should I drop that and re-insert it if we ever make such a thing?

Comment (ErikSchierboom) [reply docs/TESTS.md:37]: I think so

**Review by ErikSchierboom — APPROVED** [review]
